### DISCUSSION TITLE
New build | Plugins not working, high score not working

### DISCUSSION THREAD

**Discussion by Lyles-kauff** [runelite/runelite#17861] [Q&A] [answered]
![120050087-c0e96480-bfe1-11eb-9172-e6a4a095de5b](https://github.com/runelite/runelite/assets/168866601/78a4f925-fe16-4ec4-aa9f-93b8b6cd11dd)

for the plugins it says,
java.io.IOException: Non-OK response code: 404

EDIT:
Highscore is working

**Comment by Nightfirecat** [comment]
You are trying to fetch a plugin hub manifest which doesn't exist. This isn't really a bug because you are not running the release client.

**Reply by Lyles-kauff** [reply]
I cloned with 
git clone https://github.com/runelite/runelite.git

How can i ensure i am running the release client version? Or update the hub manifest?

**Reply by Adam-** [reply, marked as answer]
We don't provide builds of pluginhub plugins for snapshot versions. Usually you can pass -Drunelite.pluginhub.version=(current RL version)  as a VM option to load the plugins built against the release into your build, which will work if there are no ABI changes. If there are ABI changes, build the plugins yourself and place them into the sideloaded-plugins directory when running in developer mode.
